### PR TITLE
Issue #3269 - Improved CORS configuration's documentation

### PR DESCRIPTION
**Pull request by navidsh**
PR for issue #3269:
- Improved CORS config documentation

### CODE REVIEW

**Comment by navidsh** [comment]
#build

File: dev/com.ibm.ws.webcontainer.cors/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -15,19 +15,20 @@
...
-cors.desc=Refers to a cross-domain security procedure for JavaScript-based clients.
+cors.desc=Configuration for the Cross-Origin Resource Sharing (CORS).
 
 domain=Domain name
 domain.desc=The domain name represents the URL being setup with these CORS settings.
 
 allowedOrigins=Allowed origins
-allowedOrigins.desc=A comma-separated list of origins allowed to access the configured domain.
+allowedOrigins.desc=A comma-separated list of origins allowed to access the configured domain. 
+allowedOrigins.desc=A comma-separated list of origins allowed to access the configured domain. If this is set to "*", it indicates the resource is publicly accessible, with no access control checks. Also, the value of "null" indicates, the domain must not be accessible.
```
Comment (lzdanski) [inline dev/com.ibm.ws.webcontainer.cors/resources/OSGI-INF/l10n/metatype.properties]: allowedOrigins.desc=A comma-separated list of origins **that are** allowed to access the configured domain. 

allowedOrigins.desc=A comma-separated list of origins **that are** allowed to access the configured domain. If this is set to "*", it indicates the resource is publicly accessible, with no access control checks. Also, the value of "null" **indicates** the domain must not be accessible.

File: dev/com.ibm.ws.webcontainer.cors/resources/OSGI-INF/l10n/metatype.properties [hunk 2]
```diff
@@ -15,19 +15,20 @@
...
-allowedOrigins.desc=A comma-separated list of origins allowed to access the configured domain.
+allowedOrigins.desc=A comma-separated list of origins allowed to access the configured domain. 
+allowedOrigins.desc=A comma-separated list of origins allowed to access the configured domain. If this is set to "*", it indicates the resource is publicly accessible, with no access control checks. Also, the value of "null" indicates, the domain must not be accessible.  
 
 allowedMethods=Allowed methods
 allowedMethods.desc=A comma-separated list of HTTP methods allowed to be used by the origin domain when making requests to the configured domain.
 
 allowedHeaders=Allowed headers
-allowedHeaders.desc=A comma-separated list of HTTP headers allowed to be used by the origin domain when making requests to the configured domain.
+allowedHeaders.desc=A comma-separated list of HTTP headers allowed to be used by the origin domain when making requests to the configured domain. If this is set to "*", it indicates the request can include any HTTP headers.
```
Comment (lzdanski) [inline dev/com.ibm.ws.webcontainer.cors/resources/OSGI-INF/l10n/metatype.properties]: allowedHeaders.desc=A comma-separated list of HTTP headers **that are** allowed to be used by the origin domain when making requests to the configured domain. If this is set to "*", it indicates the request can include any HTTP headers.

File: dev/com.ibm.ws.webcontainer.cors/resources/OSGI-INF/l10n/metatype.properties (hunk 1 above)
Comment (lzdanski) [reply dev/com.ibm.ws.webcontainer.cors/resources/OSGI-INF/l10n/metatype.properties]: Just in case it's hard to find, you don't need a , after "null".

**Comment by navidsh** [comment]
Hi @lzdanski, I made the changes and updated the PR. Thanks

**Comment by navidsh** [comment]
#build

**Pull request merged**